doc/howtos/rdtraining/15_qwebintro.rst

```diff
@@ -0,0 +1,133 @@
+.. _howto/rdtraining/15_qwebintro:
+
+================================
+Part 15: A Brief History Of QWeb
+================================
+
+Up to now, the design of the interface of our real estate module was rather limited. Building
+a list view is straightforward since only the list of fields is necessary. The same holds true
+for the form view: despite the use of a few tags such as ``<group>`` or ``<page>``, there
+is very little to do in terms of design.
+
+However, if we want to give a unique look to our application, it is necessary to go a step
```

Comment:
And who wouldn't?
![382f4a83761e875823f30adf77308cad](https://user-images.githubusercontent.com/7571158/96463588-2775fb00-1227-11eb-84fc-ed30d1d58a73.jpg)